cmake/projects/doctest/hunter.cmake

```diff
@@ -0,0 +1,33 @@
+# cmake/projects/Example/hunter.cmake
+
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+# Load used modules
+include(hunter_add_version)
+include(hunter_cmake_args)
+include(hunter_download)
+include(hunter_pick_scheme)
+include(hunter_cacheable)
+
+# List of versions here...
+hunter_add_version(
+    PACKAGE_NAME
+    doctest
+    VERSION
+    "1.1.4"
+    URL
+    "https://github.com/piribes/doctest/archive/hunter.tar.gz"
```

Comment:
Please use stable link here, I guess it should be `https://github.com/piribes/doctest/archive/1.1.4.tar.gz`.